more/pixels/gallery.js

```diff
@@ -245,6 +261,33 @@ export function makeGallery(
     return state[rawPixel.x][rawPixel.y];
   }
 
+  // only pixels can be sold to the gallery, not use or transfer rights
+  function sellToGallery(pixelAmountP) {
+    return Promise.resolve(pixelAmountP).then(async pixelAmount => {
+      pixelAmount = pixelAssay.coerce(pixelAmount);
+      const dustAmount = pricePixelAmount(pixelAmount);
+      const dustPurse = await dustMint.mint(dustAmount);
+      const dustPaymentP = await dustPurse.withdraw(
+        dustAmount,
+        'dust for pixel',
+      );
+      // dustPurse is dropped
+      const terms = harden([dustAmount, pixelAmount]);
+      const escrowExchangeInstallationP = E(contractHost).install(
+        escrowExchangeSrc,
+      );
+      const [galleryInviteP, userInviteP] = await E(
+        escrowExchangeInstallationP,
+      ).spawn(terms);
+      const seatP = E(contractHost).redeem(galleryInviteP);
+      E(seatP).offer(dustPaymentP);
+      const dustPurseP = E(dustIssuer).makeEmptyPurse();
+      const pixelPurseP = E(pixelIssuer).makeEmptyPurse();
+      collect(seatP, pixelPurseP, dustPurseP, 'gallery escrow');
```

Comment:
[This is where the comment was intended to be.]

If I'm reading this right, the pixel proceeds are stored in pixelPurseP, which was created here, and never returned anywhere. Doesn't this lose the pixels at this point?

Reply:
That's true, those pixels don't get used anywhere. Right now, nothing will happen to them until the LRU Queue returns those coordinates and the pixel gets revoked from all of its current purses/payments (including pixelPurseP) and given to a user. 

In the future, if the gallery is selling as well, we might buy and sell out of a common purse.

Reply:
Chris just pointed out that we need to add the pixel to the back of the queue if it is colored or otherwise touched.